client/src/main/java/com/paypal/selion/internal/reports/runtimereport/MethodInfo.java

```diff
@@ -28,23 +27,24 @@
 
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
-import com.paypal.selion.annotations.MobileTest;
-import com.paypal.selion.annotations.WebTest;
 import com.paypal.selion.logger.SeLionLogger;
 import com.paypal.selion.internal.reports.model.BaseLog;
 import com.paypal.selion.internal.reports.services.ReporterDateFormatter;
 import com.paypal.test.utilities.logging.SimpleLogger;
 
+/**
+ * Used to hold test method information for reporting purposes.
+ */
 @SuppressWarnings("unused")
 class MethodInfo {
 
     private static SimpleLogger logger = SeLionLogger.getLogger();
 
-    private String suite;
-    private String test;
-    private String packageInfo;
-    private String className;
-    private String methodName;
+    private final String suite;
+    private final String test;
+    private final String packageInfo;
+    private final String className;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Perhaps 'className' could be replaced by a local variable.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1249607326/issues/source?bid=2928425&fileBranchId=2937920#l46)